src/main/scala/tech/cryptonomic/conseil/tezos/ApiOperations.scala

```diff
@@ -841,24 +833,22 @@ object ApiOperations {
     )
   }
 
+
   /**
     * Given the operation kind and the number of columns wanted,
     * return the mean (along with +/- one standard deviation) of
     * fees incurred in those operations.
     * @param filter
```

Comment:
Describe this param please and also the return value. See other function doc comments.